docs/configuration/backends/kubernetes.md

```diff
@@ -108,48 +108,39 @@ Annotations can be used on containers to override default behaviour for the whol
     Redirect to another URL for that frontend. Must be set with `traefik.frontend.redirect.replacement`.
 - `traefik.frontend.redirect.replacement: http://mydomain/$1`:
     Redirect to another URL for that frontend. Must be set with `traefik.frontend.redirect.regex`.
-- `traefik.frontend.entryPoints: http,https`  
+- `traefik.frontend.entryPoints: http,https`
     Override the default frontend endpoints.
-- `traefik.frontend.passTLSCert: true`  
+- `traefik.frontend.passTLSCert: true`
     Override the default frontend PassTLSCert value. Default: `false`.
+- `ingress.kubernetes.io/whitelist-source-range: "1.2.3.0/24, fe80::/16"`
+    A comma-separated list of IP ranges permitted for access. all source IPs are permitted if the list is empty or a single range is ill-formated.
```

Comment:
Can you use one line per sentence and begin the second sentence by an upper-case?

I'm not sure but `ill-formated` shoud not be ` ill-formatted` ?

Reply:
[You are right](https://en.wiktionary.org/wiki/formated). :-)